### ISSUE TITLE
Debug view gamepad navigation support

### ISSUE THREAD

**Issue by hagand3** [YoYoGames/GameMaker-Bugs#6823]
### Is your feature request related to a problem?

When running a game on a console or PC with a gamepad, I can't navigate through debug views due to lack of controller support. I need to be using a mouse.

### Describe the solution you'd like

As far as I know, the debug views are built using some kind of IMGUI wrapper? If that's the case, I'm fairly sure IMGUI has basic controller navigation support, but even if it doesn't or isn't applicable here, I'm hopeful it's possible to implement. 

### Describe alternatives you've considered

I've built a custom editor but not using debug views. I'd prefer to use debug views because they're awesome and quick to throw together.

**Comment by gnysek** [comment]
Seems like extended FR for what I asked - ability to at least scroll with gamepad: https://github.com/YoYoGames/GameMaker-Bugs/issues/2855 .

**Comment by hagand3** [comment]
> Seems like extended FR for what I asked - ability to at least scroll with gamepad: https://github.com/YoYoGames/GameMaker-Bugs/issues/2855 .

Upvoted yours. I could delete this one but if it's "extended", I'd rather keep it.

**Comment by gnysek** [comment]
Yeah, your is better, as it's about navigating in whole GUI, mine was for scrolling only, because it was specifically for Steam Deck, where you have cursor, but it's just hard to drag the cursor vertically perfectly.
